src/query_ctrl.ts

```diff
@@ -52,6 +54,9 @@ export class MetaQueriesQueryCtrl extends QueryCtrl {
       if (!this.target.periods) {
         this.clearPeriods();
       }
+      if (!this.target.units) {
```

Comment:
calling this `timeshiftUnit` might be better. I do understand existing code doesn't do it for periods but would love to scope it to type unless its generic. keeping in mind about stddev etc

Reply:
done